ra_aid/tools/programmer.py

```
@@ -10,12 +12,43 @@
+    """
+    # Get directory containing Python executable 
+    bin_dir = Path(sys.executable).parent
+    
+    # Check for platform-specific executable name
+    if sys.platform == "win32":
+        aider_exe = bin_dir / "aider.exe"
+    else:
+        aider_exe = bin_dir / "aider"
+        
+    if not aider_exe.exists():
+        raise RuntimeError(f"Could not find aider executable at {aider_exe}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)